src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
```

Comment:
Which module name? Like add `mobile-carousel` class to the HTML and style it here?